File: speeqeweb/webroot/scripts/local_settings.js
//domain for your website (used to build url in java scripts)
Speeqe.HTTP_DOMAIN = "tribe";
//domain for your jabber server
Speeqe.XMPP_DOMAIN = "tribe";
//domain for your multi user chat server
//multi user chat server name. This is the default chat server used if
//none is provided. If you connect to a room, the room will be
//room@Speeqe.CHAT_SERVER.
Speeqe.CHAT_SERVER = "conference.tribe.fr.murex.com";
//allows users to use the /nick command to change their muc name
Speeqe.ENABLE_NICK_CHANGE = false;
//the default chat room if none is specified. If a muc room is not
//provided, and the user connects, this will be the default room.
Speeqe.DEFAULT_CHAT_ROOM = "default@conference.tribe.com";

//the url used to proxy to your BOSH server.  Used by speeqe to
//communicate with the bosh server.
Speeqe.BOSH_URL =  "/http-bind";

//This is used to add additional help information to the help
//dialog. It will be displayed right before the instructions to close
//the dialog.
Speeqe.helpDialogHtml = "";

/* 

use this function to replace the anonymous nick selection. The default
function picks from a list of president names ,Speeqe.NAMES. See
anonymous.js for more details.

*/
/* list of anonymous nicks
Speeqe.NAMES = ["anonymous"];
Speeqe.generate_anonymous_nick = function() {

};
*/

File: speeqeweb/local_settings.py
#Domain for your website
#HTTP_DOMAIN is used to build the login url and also the username ...
HTTP_DOMAIN = "tribe"
#Domain for your jabber server
XMPP_DOMAIN = "tribe"

#names you dont want your users to have. When signing up for an
#account and the user name is one of these exact words, it will be
#rejected.
EXACT_BAD_WORDS = ['crap']
#words you don't want in your user's names.  When signing up for an
#account and the username has one of these words in it, the name will
#be rejected.
MATCH_BAD_WORDS = ['crap'] 

#where all your static documents are.
DOCUMENT_ROOT = '/var/www/speeqe/speeqeweb/webroot'
#DOCUMENT_ROOT = '/usr/local/virtualenv/askbot/lib/python2.6/site-packages/askbot/deps/speeqeweb/webroot'

#the user used to list active rooms on the website. the speeqe website
#needs a user and password to connect and run disco requests. This is
#to gather a list of active rooms from the muc component.
XMPP_USER = ''
XMPP_PASS = ''
#your multi user chat server name. used to gather a list of active rooms
XMPP_CHAT = 'conference.tribe.com'
#the name of your bosh server. Bosh is used to authenticate users to
#the website and then they are auto logged in to each chat room.
BOSH_HOST = "tribe.com"
#the port your bosh server listens on
BOSH_PORT = "7070"
#the url used to proxy to your bosh server
BOSH_URL = "/http-bind"

#multi user chat rooms listed in first on the front page.
#The rooms must exist in teh server to be displayed.
#entry is the title, followed by the room url.  You can use /room/ or
#the dns trick if you have that setup.
#since the room must exist, the room url is not used anymore.
FEATURED_ROOMS =  {'ore':'/room/ore/', 'mad':'/room/mad/', }

#where all your django templates live.  make sure to have the location
#of your themes and client.html, test_client.html in this config
#option
TEMPLATE_DIRS = (
    '/var/www/speeqe/speeqeweb/templates',
    '/var/www/speeqe/speeqeweb/webroot',
    '/usr/local/virtualenv/askbot/lib/python2.6/site-packages/askbot/deps/speeqeweb/webroot',
    '/usr/local/virtualenv/askbot/lib/python2.6/site-packages/askbot/deps/speeqeweb/templates',
    # Put strings here, like "/home/html/django_templates" or "C:/www/django/templates".
    # Always use forward slashes, even on Windows.
    # Don't forget to use absolute paths, not relative paths.
)

#allows django to serve all unknown urls as static data
SERVE_STATIC_URLS = True
#tells speeqe that you have the dns trick configured. (ie. you can use
#http://roomname.yourdomain.com). False if dns names are not configured.
DNS_ROOM_NAMES = False
